environments/aat.yml

```diff
@@ -115,4 +115,13 @@ cname:
     record: "afdverify.hmcts-aat.azurefd.net"
   - name: "afdverify.return-case-doc-ccd"
     ttl: 3600
-    record: "afdverify.hmcts-aat.azurefd.net"
\ No newline at end of file
+    record: "afdverify.hmcts-aat.azurefd.net"
+  - name: "afdverify.pcq"
+    ttl: 300
+    record: "afdverify.hmcts-aat.azurefd.net"
+  - name: "cdnverify.pcq"
```

Comment:
You don't require cdnverify here. 
cdnverify is only required for shutter pages as cdn is used for shutter applications.
We currently don't use cdn/shutter in aat environment

Reply:
Thanks for explaining. I've removed the cdnverify config.